src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -77,12 +57,12 @@ object JesBackend {
   }
 
   def workflowGcsPath(descriptor: WorkflowDescriptor): String = {
-    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, JesConf.getString("baseExecutionBucket"))
+    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, JesConf.executionBucket)
```

Comment:
If we're allowing JesConf to be mocked on line 111, let's use the mock throughout the class